### PR TITLE
batch updates of changelog items

### PR DESCRIPTION
**Pull request by notnoop**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,12 +34,17 @@ BUG FIXES:
...
  * client: Fixed a panic when running in Debian with `/etc/debian_version` is empty [[GH-7350](https://github.com/hashicorp/nomad/issues/7350)]
+ * client: Fixed a bug affecting network detection in environments that mimic the EC2 Metadata API [[GH-7509](https://github.com/hashicorp/nomad/issues/7509)]
  * client: Fixed a bug where a multi-task allocation maybe considered healthy despite a task restarting [[GH-7383](https://github.com/hashicorp/nomad/issues/7383)]
  * consul: Fixed a bug where modified Consul service definitions would not be updated [[GH-6459](https://github.com/hashicorp/nomad/issues/6459)]
  * connect: Fixed a bug where Connect enabled allocation would not stop after promotion [[GH-7540](https://github.com/hashicorp/nomad/issues/7540)]
+ * driver/docker: Fixed handling of seccomp `security_opts` option [[GH-7554](https://github.com/hashicorp/nomad/issues/7554)]
+ * driver/docker: Fixed a bug causing docker containers to use swap memory unexpectedly [[GH-7550](https://github.com/hashicorp/nomad/issues/7550)]
+ * ui: Fixed handling of UTF-8 characters and emojis in allocation log view [[GH-7470](https://github.com/hashicorp/nomad/issues/7470)]
```
Comment (notnoop) [inline CHANGELOG.md]: @DingoEatingFuzz How would you better describe the fix?  Also, should we mark it as an improvement or a bug fix?

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,12 +34,17 @@ BUG FIXES:
  * api: Fixed a panic when canonicalizing a jobspec with an incorrect job type [[GH-7207](https://github.com/hashicorp/nomad/pull/7207)]
  * api: Fixed a bug where calling the node GC or GcAlloc endpoints resulted in an error EOF return on successful requests [[GH-5970](https://github.com/hashicorp/nomad/issues/5970)]
  * api: Fixed a bug where `/client/allocations/...` (e.g. allocation stats) requests may hang in special cases after a leader election [[GH-7370](https://github.com/hashicorp/nomad/issues/7370)]
+ * cli: Fixed a bug where `nomad agent -dev` fails on Windows [[GH-7435](https://github.com/hashicorp/nomad/issues/7435)]
```
Comment (jazzyfresh) [inline CHANGELOG.md:37]: This PR appears to be for CSI docs, not a cli fix?

Comment (jazzyfresh) [reply CHANGELOG.md:37]:
```suggestion
 * cli: Fixed a bug where `nomad agent -dev` fails on Windows [[GH-7534](https://github.com/hashicorp/nomad/pull/7534)]
```

File: CHANGELOG.md (hunk 1 above)
Comment (DingoEatingFuzz) [reply CHANGELOG.md:47]: The precise issue is a lack of multibyte unicode characters (which includes emoji) in the native browser lib (window.atob).

I would consider it to be a bug fix.

There is also a second PR for this same changelog entry (replacing the community contribution with one that matches the Exec implementation): https://github.com/hashicorp/nomad/pull/7551

Thanks for adding this to the log!

Comment (jazzyfresh) [reply CHANGELOG.md]: updated

**Comment by jazzyfresh** [comment]
@shoenig please make changelog commits here

**Pull request merged**